packages/build-utils/src/lambda.ts

```diff
@@ -267,5 +269,8 @@ export async function getLambdaOptionsFromFunction({
     }
   }
 
-  return {};
+  const project = new Project();
+  const staticConfig = getConfig(project, sourceFile);
+
+  return staticConfig ? staticConfig : {};
```

Comment:
I guess this function is for extracting lambda parameters from framework builders, not parsing the script again. Perhaps we parse the script somewhere else?